Documentatie/EnergielabelApi-validaties.md

```diff
@@ -103,28 +104,27 @@ Onderstaande validaties worden allemaal en in willekeurige volgorde uitgevoerd.
+|:------------------|:------------------------------------------|:----------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+|	Toevoegen	|	<ul><li>Vergunningsaanvraag</li><li>Oplevering</li><li>Bestaand</li></ul>	|<ul><li> Toevoegen is alleen mogelijk wanneer er op de opnamedatum voor geen enkele van de opnameadressen al een registratie is met dezelfde scope en gebouwklasse.</li><li> Toevoegen voor de status ‘Vergunningsaanvraag’ is alleen mogelijk wanneer één van de blokken ‘BAGIdentification’ of ‘ProvisionalIdentification’ gekozen is. </li><li> Toevoegen voor de status ‘Vergunningsaanvraag’ is niet mogelijk met een ‘provisionalid’. </li><li> Toevoegen voor woningbouw met een detailaanduiding is alleen mogelijk met rechten. Deze kunt u aanvragen in EP-Online. </li></ul>
+|	Vervangen		|	<ul><li>Vergunningsaanvraag</li><li>Oplevering</li><li>Bestaand</li></ul>	|	<ul><li>Vervangen is alleen mogelijk wanneer er op de opnamedatum al een registratie is met dezelfde scope en gebouwklasse, waarbij alle adressen overeen komen met de initiële registratie.</li><li>Vervangen is alleen mogelijk wanneer de rekenmethodiek versie overeen komt met de versie die gebruikt is in de initiële registratie.</li><li>Vervangen is niet mogelijk wanneer een of meer van de gebouwen binnen de registratie ingetrokken is.</li><li>Vervangen is alleen mogeljik door de registratieadviseur die de initiële registratie gedaan heeft.</li><li>Vervangen van status ‘Vergunningsaanvraag’ is mogelijk zonder rechten binnen 60 dagen vanaf de originele registratiedatum. Hierna zijn rechten nodig. Deze kunt u aanvragen in EP-Online.</li><li>Vervangen van status ‘Oplevering’ of status ‘Bestaand’ is mogelijk zonder rechten binnen 7 dagen vanaf de originele registratiedatum. Hierna zijn rechten nodig. Deze kunt u aanvragen in EP-Online.</li></ul>
+|	Uitbreiden		|	<ul><li>Oplevering</li><li>Bestaand</li></ul>		|	<ul><li>Uitbreiden is alleen mogelijk wanneer er op de opnamedatum al een registratie is met dezelfde scope en gebouwklasse, waarbij alle adressen overeen komen met de initiële registratie.</li><li> Uitbreiden is alleen mogelijk wanneer de meeste recente rekenmethodiek versie wordt gebruikt.</li><li> Uitbreiden is alleen mogelijk voor status ‘Oplevering’ en status ‘Bestaand’. </li><li> Uitbreiden is alleen mogelijk wanneer het adres/de adressen waarmee uitgebreid wordt/worden nog geen recenter(e) registratie heeft/hebben. </li><li> Uitbreiden voor woningbouw met een detailaanduiding is alleen mogelijk met rechten. Deze kunt u aanvragen in EP-Online. </li></ul> 
```

Comment:
Het derde bullet (uitbreiden alleen mogelijk voor status oplevering en bestaand) hoeft niet expliciet uitgeschreven te worden, want het staat ook al in de kolom hiervoor. Bij andere acties leggen we dat ook niet uit.